src/components/ConnectionFlowDialogs.tsx

```diff
@@ -84,6 +85,9 @@ const ConnectionDialogs = () => {
 
   const dialogCommonProps = { isOpen, onClose };
 
+  // TODO: Use EditCodeDialog state to prevent standard connection flow
+  // dialogs from being displayed over MakeCode, whether or not a MakeCode
+  // download is in progress.
```

Comment:
Flagging todo - subsequent PR where the EditCodeDialog disclosure state is already pulled up.

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/301/commits/643ea057757636130ce85d266ea5e065799d3588